### ISSUE TITLE
All Deals Page: "SHOW MY DEALS ONLY" toggle should be automatically enabled after clicking it and connecting the wallet

### ISSUE THREAD

**Issue by BartuCulha** [PrimeDAO/prime-deals-dapp#427]
If it's possible, we should toggle on the button automatically after the user clicks it and conencts their wallet
Result:
https://images.zenhubusercontent.com/394656771/c596f6e3-649b-43af-9a1d-96bf6d398122/desktop_1647867848149.mp4
Expected:
After connecting wallet, show my deals only toggle is enabled

**Comment by vipergtsrz** [comment]
@BartuCulha I'm not sure if this is possible with how it's currently built because the ethereumService.ensureConnected() method that we use to connect the wallet isn't an async method and I can't pass it a callback function currently. So the code currently isn't built to pass back if the wallet got connected and then toggle it on. It would require some additional work and regression testing to make sure it works properly. @dkent600 Any thoughts?

**Comment by dkent600** [comment]
> Any thoughts?

Agreed.  We have never done things the way @BartuCulha is requesting.  In any case, our plan was to show a Connect button instead of the toggle when they aren't connected.  Dunno how that changed.  Perhaps folks gave up trying to come up with good text for the button.  But it would address the problem and be better UX.

**Comment by vipergtsrz** [comment]
@dkent600 I had it developed as a "Connect" button originally, but in the final page review we had, it got changed back to the toggle with with tooltip. I can change it back if you'd like.

**Comment by dkent600** [comment]
@BartuCulha With this we are going back to our original plan to have a Connect button.  This doesn't completely address your wish, but it is consistent with everywhere else in the Prime suite, and is better UX.  Technically it is difficult to do what you're asking for.
